cli_li3ds/import_platform.py

```diff
@@ -0,0 +1,87 @@
+import logging
+
+from cliff.command import Command
+
+from . import api
+
+
+class ImportPlatform(Command):
+    """ import a sample platform configuration for stéréopolis
+    """
+
+    log = logging.getLogger(__name__)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_parser(self, prog_name):
+        self.log.debug(prog_name)
+        parser = super().get_parser(prog_name)
+        api.add_arguments(parser)
+        return parser
+
+    def take_action(self, parsed_args):
+        server = api.ApiServer(parsed_args, self.log)
+        objs = api.ApiObjs(server)
+
+        self.log.info('Importing platform configuration sample')
+
+        lidar = api.Sensor({
+            'name': 'lidar',
+            'type': 'lidar',
+            'description': 'Imported from cli-li3ds'})
+
+        ins = api.Sensor({
+            'name': 'ins',
+            'type': 'ins',
+            'description': 'Imported from cli-li3ds'})
+
+        camera_group = api.Sensor({
+            'name': 'cameraMetaData.json',
+            'type': 'group'})
+
+        ref_camera_group = api.Referential(
+            camera_group,
+            {'name': 'base'})
+
+        ref_lidar = api.Referential(
+            lidar,
+            {'name': 'ref lidar cartesian'})
```

Comment:
Do you mind removing "ref" here in the referential name? It's pattern we haven't used elsewhere.